src/AzureAppConfigurationImpl.ts

```diff
@@ -342,18 +340,18 @@ export class AzureAppConfigurationImpl implements AzureAppConfiguration {
         return key;
     }
 
-    async #getConfigurationSettingWithTrace(configurationSettingId: ConfigurationSettingId, customOptions?: GetConfigurationSettingOptions): Promise<GetConfigurationSettingResponse | undefined> {
+    /**
+     * Get a configuration setting by key and label. If the setting is not found, return undefine instead of throwing an error.
+     */
+    async #getConfigurationSetting(configurationSettingId: ConfigurationSettingId, customOptions?: GetConfigurationSettingOptions): Promise<GetConfigurationSettingResponse | undefined> {
         let response: GetConfigurationSettingResponse | undefined;
         try {
-            const options = { ...customOptions ?? {} };
-            if (this.#requestTracingEnabled) {
-                options.requestOptions = {
-                    customHeaders: {
-                        [CORRELATION_CONTEXT_HEADER_NAME]: createCorrelationContextHeader(this.#options, this.#isInitialLoadCompleted)
-                    }
-                }
-            }
-            response = await this.#client.getConfigurationSetting(configurationSettingId, options);
+            response = await getConfigurationSettingWithTrace({
+                requestTracingEnabled: this.#requestTracingEnabled,
+                initialLoadCompleted: this.#isInitialLoadCompleted,
+                appConfigOptions: this.#options
+            }, this.#client, configurationSettingId, customOptions);

```

Comment:
Can we add a linting rule to unify 
![image](https://github.com/Azure/AppConfiguration-JavaScriptProvider/assets/141655842/fdcddd3c-5d04-4b10-ae35-d46e8e222e99)
and
![Uploading image.png…]()

Reply:
well it's code style flavor, whether the bracket starts in a new line.  I prefer K&R style and it's more commonly seen is JS world. 
BTW some related topics from internet. https://stackoverflow.com/a/3218860

Reply:
unified styles in 3f2a134